examples/schema-test/main.go

```diff
@@ -34,13 +124,14 @@ func (s *Strct) Annotate(a resource.Annotator) {
 func main() {
 	println(reflect.TypeOf((*Enum)(nil)).Elem().String())
```

Comment:
Why do we do this? I think it will invalidate the provider, since we assume that providers print their address first?

Reply:
Ah, meant to delete that. Also I wasn't aware of that.